test.js

```diff
@@ -137,7 +165,7 @@ test('support `Object.create(null)` objects', t => {
 test('support `Object.create(null)` targets', t => {
 	var obj = Object.create(null);
 	obj.foo = true;
-	t.same(fn(obj, {bar: false}), {foo: true, bar: false});
+	t.same(fn({}, obj, {bar: false}), {foo: true, bar: false});
```

Comment:
@coderhaoxin Thank you for mentioning that. You were right, this test had to be repaired (e252fa2dec757a6c608e242ca49ca060f2439ad2).